apps/api/src/app/organization/usecases/create-organization/create-organization.usecase.ts

```diff
@@ -53,7 +56,15 @@ export class CreateOrganization {
       })
     );
 
-    await this.createEnvironmentUsecase.execute(
+    await this.createNovuIntegrations.execute(
```

Comment:
Creating for dev and prod separate to make it easier make it into events in the future.